src/components/GoalPreview/GoalPreview.tsx

```diff
@@ -305,12 +303,14 @@ const GoalPreview: React.FC<GoalPreviewProps> = ({ preview, onClose, onDelete })
                         </CardInfo>
 
                         <CardComment>
-                            {description ? (
-                                <Md>{description}</Md>
-                            ) : (
-                                <Text size="s" color={gray7} weight="thin">
-                                    {tr('No description provided')}
-                                </Text>
+                            {nullable(description, (d) =>
```

Comment:
This code is incorrect. In case of description===undefined nothing will be rendered